site/content/blog/2022-09-10-zap-extender-scripts/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Hacking ZAP - ZAP Extender Scripts"
+description: "Use ZAP as a web server, subscribe to internal ZAP events, and more!"
+summary: >
+    An overview of ZAP Extender scripts with examples.
+    Use ZAP as a web server, subscribe to internal ZAP events, and more!
+images:
+- https://www.zaproxy.org/blog/2022-09-10-zap-extender-scripts/images/_zapbot-extender-script.png
+type: post
+tags:
+- hacking
+- scripts
+- youdontknowzap
+date: "2022-09-10"
+addSocialPreview: true
+authors:
+- akshath
+---
+
+
+I recently learnt about ZAP Extender scripts and I wish I knew about them sooner!
```

Comment:
I'd drop "ZAP" before all mentions (after this one), it's clear that they are ZAP scripts.